### ISSUE TITLE
[Feature Request] Allow to open different documents in multiple tabs / instances

### ISSUE THREAD

**Issue by 7flash** [excalidraw/excalidraw#5416]

**Comment by jimkberry** [comment]
It used to work to open the tabs/windows as "private", at least in Brave - tho I'm pretty sure in Chrome, too. I wonder when that stopped working, and why? 

I haven't used excalidraw in a while and am about to need it again and this will be a real problem if I can't find another workaround,

**Comment by jimkberry** [comment]
Ok, I see what has happened - longer ago that I thought! :-)

Anyway, I also have seen the discussions regarding the consequences of separate tabs having separate local storage keys and why folks aren't comfortable making tab states atomic. I think I see a way to make everyone happy, tho - or at least to make it do what I (and the OP, I presume) want without breaking the expected tab/drawing sync behavior:

- Introduce the idea of "Sandbox Mode" for an Excalidraw session (tab.) Once selected, Sandbox is permanent and irrevocable for that particular session. A session in sandbox mode has a visible "Sandbox Mode" bug displayed in the UI, just to make sure folks know.
- When set, sandbox mode simply causes the app to use `window.sessionStorage` in all places where it currently uses `window.localStorage`.  The 2 APIs are identical, but `sessionStorage` only persists as long as the session and is NOT shared across tabs/windows.

I think that's all of it. I expect that that immediately after the  switch the current state will have to be saved (to sessionStorage this time) before proceeding. - I'd like it best if when I went into sandbox mode the tab stayed the way it was when I entered it. There's no cleanup needed since you can't leave the mode. 

There are only 3 source files in the app with references to `window.localStorage` and one in the tests.

I think I'll take a crack at it in codesandbox and see what happens.

**Comment by jimkberry** [comment]
Yeah, ok, so not surprisingly I've gotten tied up with other stuff and haven't done anything code-wise for this.

However: I *did* find a workaround which allows you to have multiple simultaneous discrete sessions: just create yourself 3 or 4 browser "profiles" - in Brave anyway setting them up is pretty trivial, as is switching between them and even saving a desktop shortcuts to start a new browser windows with each of them.

This works well enough that I'm not going to try anything further. It also has the added advantage that each separate profile's data is persistent so you do things like having a profile dedicated to a particular "atlas" document and/or set of related libraries - ready to fire up at a moment's notice to use as a source to copy from into whatever you are working on in another session.
